src/Uno.UWP/Devices/Sensors/Helpers/INativeHingeAngleSensor.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.Devices.Sensors.Helpers;
+using Windows.Devices.Sensors;
+using Windows.Foundation;
+
+namespace Uno.Devices.Sensors
+{
+    public interface INativeHingeAngleSensor
```

Comment:
All the WinRT apis are backed by an internal interface. Those are not yet exported by the Sync generator, but will soon be. For example, this one has this:

```csharp
namespace Windows.Devices.Sensors
{
	[ContractVersion(typeof(UniversalApiContract), 458752)]
	[ExclusiveTo(typeof(HingeAngleSensor))]
	[GuidAttribute(3922968066, 49119, 17279, 140, 41, 136, 199, 115, 147, 211, 9)]
	internal interface IHingeAngleSensor
	{
		[RemoteAsync]
		IAsyncOperation<HingeAngleReading> GetCurrentReadingAsync();

		string DeviceId { get; }
		double MinReportThresholdInDegrees { get; }
		double ReportThresholdInDegrees { get; set; }

		event TypedEventHandler<HingeAngleSensor, HingeAngleSensorReadingChangedEventArgs> ReadingChanged;
	}
}
```

Reply:
I was thinking that it still might be worth keeping as a separate interface to make it clear that it is a Uno extension. Plus I would have to add public constructor for HingeAngleReading (so the implementation can trigger the event)

Reply:
Fair points. The case for the public constructors will most probably have to be dealt with in the context of a more general extension mechanism. Let's keep it that way for now then.